config.json

```diff
@@ -1830,6 +1830,22 @@
         ],
         "difficulty": 6,
         "topics": []
+      },
+      {
+        "slug": "poker",
+        "name": "Poker",
+        "uuid": "e855d32e-9d5d-4342-a3e9-e70cf2329499",
+        "practices": [],
+        "prerequisites": [
+          "loops",
+          "arrays",
+          "array-transformations",
+          "objects",
+          "conditionals",
+          "strings"
+        ],
+        "difficulty": 4,
```

Comment:
Difficulty seems way to low. :thinking: Maybe 7 like Zebra Puzzle would be good fit.